contracts/UFragmentsPolicy.sol

```diff
@@ -57,18 +55,25 @@ contract UFragmentsPolicy is Ownable {
     // The number of rebase cycles since inception
     uint256 public epoch;
 
-    uint256 private constant RATE_DECIMALS = 18;
-
-    uint256 private constant TARGET_RATE = 1 * 10**RATE_DECIMALS;
+    // TODO: Explain this parameter
+    uint256 public targetRate;
```

Comment:
Do we need to have targetRate in storage? Sending it as a parameter to computeSupplyDelta is sufficient.

Reply:
I agree. 👍 
If off-chain services need this value, they can compute it using the data from the CPI oracle and the initial cpi or rate.